README.md

```diff
@@ -7,4 +7,59 @@ Course in Ecology and Evolutionary Biology
 [![Build Status](https://travis-ci.org/UofTCoders/rcourse.svg?branch=master)](https://travis-ci.org/UofTCoders/rcourse)
 [![DOI](https://zenodo.org/badge/97400494.svg)](https://zenodo.org/badge/latestdoi/97400494)
 
+
+## Description
+
+Quantitative Methods in R for Biology is a course aimed at undergraduates at a third year level or above. 
+The course covers statistics and data analysis for ecology and reproducible quantitative methods in R. 
+
+Statistical analysis, modelling, simulation, and data analysis are essential skills for applying 
+ecology concepts to data. This course is designed to meet a growing demand for reproducible, openly accessible, 
+analytically thorough, and well-documented science. Students will learn to develop ecological population models, 
+analyze data, and document their research using the R programming language. No prior programming experience is required.
+
```

Comment:
Maybe add something about how students will learn to clean, manipulate, and visualize "big data", which is increasingly generated and used in a range of scientific disciplines.